gn3/settings.py

```diff
@@ -12,6 +12,6 @@
 TMPDIR = os.environ.get("TMPDIR", tempfile.gettempdir())
 
 # SQL confs
-SQLALCHEMY_DATABASE_URI = "mysql://kabui:1234@localhost/test"
+SQL_URI = os.environ.get("SQL_URI", "mysql://kabui:1234@localhost/db_webqtl")
```

Comment:
How about, instead of using an SQL_URI, we just set the connection params. WDYT?

Reply:
I think having env variables for each might be the only alternative cause the order matter